dockerfiles/centos_7

```diff
@@ -1,12 +1,10 @@
-ARG BASE_IMAGE
-FROM ${BASE_IMAGE}centos:7
+FROM centos:7
```

Comment:
Why did you replace it? To make Dockerfile usable without makefile?

This variable is set in https://github.com/tarantool/docker/blob/master/.gitlab.mk#L18. If we don't need it, it should be removed from Makefile as well.

And it had the `${ARCH}` part, where did it go?

Reply:
You are right. Variable ARCH, BASE_IMAGE etc we don't use. Ok I will remove it.